### PR TITLE
Add hemi UI

### PR DESCRIPTION
**Pull request by leanazulyoro**
This fork of uniswap's interface adds support for Hemi Sepolia.

- Cleans up some elements that won't be used: 
  - navbar "explore" and "nfts" items, and dropdown menu
  - searchbar on top for tokens and NFT collections
  - portfolio in wallet drawer
- It also removes chains from the chainSelector, leaving only "Hemi Sepolia" for now.
- Finally, it changes the logo and favicon for Hemi's.

### CODE REVIEW

File: apps/web/src/components/Logo/ChainLogo.tsx [hunk 1]
```diff
@@ -81,6 +82,12 @@ export function getChainUI(chainId: ChainId, darkMode: boolean): ChainUI | undef
         bgColor: '#0052FF33',
         textColor: '#0052FF',
       }
+      case ChainId.HEMI_SEPOLIA: 
+      return {
+        Symbol: hemi,
+        bgColor: '#FF042033',
+        textColor: '#FF0420',
```
Comment (gndelia) [inline apps/web/src/components/Logo/ChainLogo.tsx]: these are copied from Optimism, but we may want to define the proper colors. (maybe pinging Dean)

File: apps/web/src/constants/chainInfo.ts [hunk 2]
```diff
@@ -242,6 +243,26 @@ const CHAIN_INFO: ChainInfoMap = {
...
+    // TODO define json list
+    defaultListUrl: BASE_LIST,
+    docs: 'https://docs.hemi.xyz',
+    explorer: 'https://testnet.explorer.hemi.network',
+    // TODO define
+    infoLink: 'https://docs.hemi.xyz/',
+    label: 'Hemi Sepolia',
+    // TODO define
+    statusPage: 'https://optimism.io/status',
+    // TODO define
```
Comment (gndelia) [inline apps/web/src/constants/chainInfo.ts]: we should create an issue to confirm/update all of these

File: apps/web/src/components/NavBar/ChainSelector.tsx [hunk 3]
```diff
@@ -146,7 +146,6 @@ export const ChainSelector = ({ leftAlign }: { leftAlign?: boolean }) => {
     width: 20,
     color: theme.neutral2,
   }
-
```
Comment (gndelia) [inline apps/web/src/components/NavBar/ChainSelector.tsx]: maybe we should avoid adding these which could potentially add rebasing conflicts in the future (if we ever rebase)

File: apps/web/src/constants/chainInfo.ts (hunk 2 above)
Comment (gabmontes) [reply apps/web/src/constants/chainInfo.ts]: TODOs with a ref to a GitHub issue > GitHub issues > TODOs throughout the code

File: apps/web/src/components/Logo/ChainLogo.tsx (hunk 1 above)
Comment (leanazulyoro) [reply apps/web/src/components/Logo/ChainLogo.tsx]: Changed it to match colors from hemi's site, but we should wait to see Dean's uniswap design to style more accordingly.
https://github.com/hemilabs/interface/pull/1/commits/a5a5a47d4bcfde0df5da4a3e46ddc6b36e953935

File: apps/web/src/constants/chainInfo.ts (hunk 2 above)
Comment (leanazulyoro) [reply apps/web/src/constants/chainInfo.ts]: created the issue and referenced it from the TODO: https://github.com/hemilabs/interface/pull/1/commits/447c41cd5e18dbf107c45eea506c98090550a543

File: apps/web/src/components/NavBar/ChainSelector.tsx (hunk 3 above)
Comment (leanazulyoro) [reply apps/web/src/components/NavBar/ChainSelector.tsx]: Re-added here: https://github.com/hemilabs/interface/pull/1/commits/447c41cd5e18dbf107c45eea506c98090550a543

**Review by gndelia — DISMISSED** [review]

File: apps/web/src/components/Logo/ChainLogo.tsx (hunk 1 above)
Comment (gndelia) [reply apps/web/src/components/Logo/ChainLogo.tsx]: great, yeah but let's go with it for the time being

**Review by gndelia — DISMISSED** [review]

**Review by gndelia — APPROVED** [review]

**Review by gabmontes — APPROVED** [review]
uACK

**Pull request merged**